src/osx/cocoa/dnd.mm

```diff
@@ -318,6 +318,9 @@ - (NSDragOperation)code
 
 - (NSDragOperation)draggingSession:(nonnull NSDraggingSession *)session sourceOperationMaskForDraggingContext:(NSDraggingContext)context
 {
+    wxUnusedVar( session );
+    wxUnusedVar( context );
```

Comment:
Like with most other changes in this commit already: there should be no spaces around the parameters here (and elsewhere in this commit).